src/utils/validateInputWithRegex.test.tsx

```diff
@@ -0,0 +1,30 @@
+import { validate, validateAndCleanId } from "./validateInputWithRegex";
+import { EnvVersionError } from "../services/envVersion";
+
+describe("validate", () => {
+  const alphanumericRegex = "^[a-zA-Z0-9-_ ]+$";
```

Comment:
This example regex also includes `-`, `_` and ` `, do we want to test for these too?